CONTRIBUTION_LADDER.md

```diff
@@ -95,7 +95,7 @@ the maintainers.
 * Manage porter-* repositories
```

Comment:
These are out of date and should be fixed as well since we no longer have porter-* repositories and teams. It should be "manage the getporter repositories" and "manage the getporter teams"